app/src/main/java/ch/epfl/polycrowd/authentification/LoginActivity.java

```diff
@@ -77,6 +74,7 @@ private UserHandler successHandler(){
             //Successful login redirects to front page
 
             PolyContext.setCurrentUser(user);
+            PolyContext.getUserLocator().setRegisteredUser(user);
```

Comment:
Why have the currentUser stored in two places?

Reply:
I imagine that the realtime database might store some true or false field for whether the user is logged in or not, and if so, possibly their user-id.